ucp/_libs/ucp_py.pyx

```diff
@@ -332,47 +332,47 @@ cdef class ucp_py_ep:
             else:
                 nbytes = len(msg)
 
-        internal_msg = ucp_msg(buf_reg, name=name, length=nbytes)
-        internal_msg.ucp_ep = self.ucp_ep
+        internal_msg = UCPMessage(buf_reg, name=name, length=nbytes)
+        internal_msg.ep = self.ep
 
-        internal_msg.ctx_ptr = ucp_py_ep_send_nb(self.ucp_ep, internal_msg.buf, nbytes)
+        internal_msg.ctx_ptr = ucp_py_ep_send_nb(self.ep, internal_msg.buf, nbytes)
         internal_msg.comm_len = nbytes
         internal_msg.ctx_ptr_set = 1
 
         fut = handle_msg(internal_msg)
         return fut
 
     def close(self):
-        if -1 == ucp_py_put_ep(self.ucp_ep):
+        if -1 == ucp_py_put_ep(self.ep):
             raise NameError('Failed to close endpoint')
 
-cdef class ucp_listener:
+cdef class UCPListener:
```

Comment:
Similar question - Should it be `UCPListener` or just `Listener`?

Reply:
Sounds good to me